### DISCUSSION TITLE
GPS/Location with PlayCover?

### DISCUSSION THREAD

**Discussion by fabiosolive** [PlayCover/PlayCover#1372] [General]
Hi. Do you have any plans to add the feature to use Mac GPS/location on PlayCover? I know Macs are not equipped with the hardware, but maybe emulate using MacOS location services? Thanks!

**Comment by XuYicong** [comment]
That's not important enough compared with all those bugs we have to fix. It's a good idea but just lacks people who implement it.

**Comment by Depal1** [comment]
For further reference, there is a Swift package for spoofing iPhone Simulator location.
https://github.com/Schlaubischlump/LocationSpoofer
https://github.com/Schlaubischlump/LocationSimulator
